app/assets/javascripts/discourse/tests/integration/components/form-template-field/wrapper-test.js

```diff
@@ -89,7 +89,7 @@ module(
             id: 1,
             name: "Bug Reports",
             template:
-              '- type: checkbox\n  choices:\n    - "Option 1"\n    - "Option 2"\n    - "Option 3"\n  attributes:\n    label: "Enter question here"\n    description: "Enter description here"\n    validations:\n      required: true',
+              '- type: checkbox\n  id: options\n  choices:\n    - "Option 1"\n    - "Option 2"\n    - "Option 3"\n  attributes:\n    label: "Enter question here"\n    description: "Enter description here"\n    validations:\n      required: true',
```

Comment:
Curious, is this change needed?

Reply:
There's an Ember assertion on the `get` helper and it fails the test when the id is empty [here](https://github.com/discourse/discourse/blob/72b2b9238671d5520662ce9395ea183ef26958f6/app/assets/javascripts/discourse/app/components/form-template-field/wrapper.gjs#L36), because `The key provided to get must be a string or number, you passed undefined`. As ID is a mandatory field, I thought it was enough to make tests reflect this premise, but I can do some extra validation if you believe it to be helpful.